_api-reference/snapshots/create-repository.md

```diff
@@ -74,6 +75,7 @@ Request field | Description
 `max_restore_bytes_per_sec` | The maximum rate at which snapshots restore. Default is 40 MB per second (`40m`). Optional.
 `max_snapshot_bytes_per_sec` | The maximum rate at which snapshots take. Default is 40 MB per second (`40m`). Optional.
 `readonly` | Whether the repository is read-only. Useful when migrating from one cluster (`"readonly": false` when registering) to another cluster (`"readonly": true` when registering). Optional.
+`remote_store_index_shallow_copy` | Boolean | Whether or not the current snapshot is a full copy or shallow copy. 
```

Comment:
here as well: `Whether or not the snapshot of remote store enabled indices be captured as shallow copy.' also, we can add that the default value of this will be False.